tests/fixtures/dnadiff/targets/delta/OP073605_vs_MGV-GENOME-0266457.delta

```diff
@@ -1,4 +1,4 @@
-/Users/pc40583/repositories/pyani-plus/tests/generate_fixtures/../fixtures/viral_example/OP073605.fasta /Users/pc40583/repositories/pyani-plus/tests/generate_fixtures/../fixtures/viral_example/MGV-GENOME-0266457.fna
+/Users/angelikakiepas/Desktop/pyani-plus/tests/generate_fixtures/../fixtures/viral_example/OP073605.fasta /Users/angelikakiepas/Desktop/pyani-plus/tests/generate_fixtures/../fixtures/viral_example/MGV-GENOME-0266457.fna
```

Comment:
I wouldn't have bothered checking in these meaningless changes to the delta files - it just adds noise to the commit history.

Reply:
Fair point. Is there a way to keep other changes, but at the same time disregard changes to the delta files?

Reply:
This has been resolved in https://github.com/pyani-plus/pyani-plus/pull/118/commits/a0fde5448eda119f7dc7dacf5c3ba3fdb0eb0ba5.